ebi_eva_common_pyutils/spring_properties.py

```diff
@@ -121,6 +123,95 @@ def get_accessioning_properties(self, *, instance=None, target_assembly=None,  f
                 'parameters.outputVcf': output_vcf
             },
         )
+    def _common_eva_pipeline_properties(self, opencga_path, read_preference='se'):
+        mongo_host, mongo_user, mongo_pass = get_primary_mongo_creds_for_profile(
+            self.maven_profile, self.private_settings_file)
+        counts_url, counts_username, counts_password = get_count_service_creds_for_profile(
+            self.maven_profile, self.private_settings_file)
+        variant_url, variant_user, variant_pass = get_job_tracker_creds_for_profile(self.maven_profile,
+                                                                                   self.private_settings_file)
+
+        return {
+            'spring.profiles.active': 'production,mongo',
+            'spring.profiles.include': 'variant-writer-mongo,variant-annotation-mongo',
+
+            'spring.data.mongodb.authentication-database': 'admin',
+            'spring.data.mongodb.host': mongo_host,
+            'spring.data.mongodb.password': mongo_pass,
+            'spring.data.mongodb.port': 27017,
+            'spring.data.mongodb.username': mongo_user,
+            'spring.data.mongodb.authentication-mechanism': 'SCRAM-SHA-1',
+
+            'job.repository.driverClassName': 'org.postgresql.Driver',
+            'job.repository.url': variant_url,
+            'job.repository.username': variant_user,
+            'job.repository.password': variant_pass,
+
+            'eva.count-stats.url': counts_url,
+            'eva.count-stats.username': counts_username,
+            'eva.count-stats.password': counts_password,
+
+            'app.opencga.path': opencga_path,
+            'config.restartability.allow': 'false',
+            'config.db.read-preference': read_preference,
+
+            'logging.level.embl.ebi.variation.eva': 'DEBUG',
+            'logging.level.org.opencb.opencga': 'DEBUG',
+            'logging.level.org.springframework': 'INFO',
+
+            'spring.main.allow-bean-definition-overriding': 'true',
+            }
+
```

Comment:
Some observations:

* Suggest re-using common properties as much as possible (for ex: mongo properties) and only put in the additional properties here.
* If you feel that there are some properties here that should belong in the common properties and might benefit other jobs as well, suggest adding it [here](https://github.com/EBIvariation/eva-common-pyutils/blob/2c7d5dbdb8567f4c602f6db8bdfe4dca2c771350/ebi_eva_common_pyutils/spring_properties.py#L55)

Reply:
Reformatted the code to remove duplicates and reuse common properties